.github/workflows/build.yml

```diff
@@ -113,10 +132,11 @@ jobs:
     - uses: actions/checkout@v2
 
     - uses: actions/cache@v1
-      name: Cache cabal store
+      name: Cache ~/.cabal/store
+      id: cache-store
       with:
         path: ~/AppData/Roaming/cabal/store
-        key: ${{ runner.os }}-cabal-store
+        key: ${{ hashFiles ('**\hscli.cabal') }}-${{ runner.os }}-cabal-store
```

Comment:
does this have to be prefixed with `**\`? if so, that's annoying

Reply:
Windows is the only one I haven't been able to get to work. This way it does find a filehash but I'm not 100% confident that it is for `hscli.cabal`. I had a lot of issues trying to get `AppData/Roaming/hsci.cabal` to work. I'm putting this on hold for now. Mac and Linux being cached are helpful enough to get the tests running faster.